client/src/modules/home/index.tsx

```diff
@@ -22,7 +22,11 @@ type User = NonNullable<AuthContextType['user']>;
 
 const Welcome = ({ user }: { user: User }) => {
   return (
-    <Flex alignItems={'center'} justifyContent="space-between">
+    <Flex
+      alignItems={'center'}
+      justifyContent="space-between"
+      paddingTop="20px"
```

Comment:
I have selected the wrong line 😅

```suggestion 
      marginTop="20px"
```